alf/algorithms/hypernetwork_algorithm.py

```diff
@@ -230,12 +235,14 @@ def sample_parameters(self, noise=None, num_particles=None, training=True):
         """Sample parameters for an ensemble of networks. 
         
         Args:
-            noise (Tensor): input noise to self._generator.
-            num_particles (int): number of sampled particles.
+            noise (Tensor): input noise to self._generator. Default is None.
+            num_particles (int): number of sampled particles. Default is None.
+                If both noise and num_particles are None, self._num_particles
```

Comment:
``num_particles`` provided to the constructor will be used ...

Reply:
updated.